e2e/linter/src/linter.test.ts

```diff
@@ -572,10 +569,61 @@ describe('Linter', () => {
       ]);
       expect(e2eEslint.overrides[0].extends).toBeUndefined();
       expect(e2eEslint.overrides[1].extends).toBeUndefined();
-      expect(libEslint.overrides[0].extends).toEqual([
-        'plugin:@nrwl/nx/angular',
-        'plugin:@angular-eslint/template/process-inline-templates',
+      expect(libEslint.overrides[0].extends).toBeUndefined();
+    });
+
+    it('(Node standalone) should set root project config to app and e2e app and migrate when another lib is added', () => {
+      const myapp = uniq('myapp');
+      const mylib = uniq('mylib');
+
+      runCLI(
+        `generate @nrwl/node:app ${myapp} --rootProject=true --no-interactive`
+      );
+
+      let rootEslint = readJson('.eslintrc.json');
+      let e2eEslint = readJson('e2e/.eslintrc.json');
+      expect(() => checkFilesExist(`.eslintrc.base.json`)).toThrow();
+
+      // should directly refer to nx plugin
+      expect(rootEslint.plugins).toEqual(['@nrwl/nx']);
+      expect(e2eEslint.plugins).toEqual(['@nrwl/nx']);
+      // should only extend framework plugin
+      expect(e2eEslint.extends).toEqual([]);
+      // should have plugin extends
+      expect(rootEslint.overrides[0].extends).toEqual([
+        'plugin:@nrwl/nx/typescript',
+      ]);
+      expect(rootEslint.overrides[1].extends).toEqual([
+        'plugin:@nrwl/nx/javascript',
+      ]);
+      expect(e2eEslint.overrides[0].extends).toEqual([
+        'plugin:@nrwl/nx/typescript',
+      ]);
+      expect(e2eEslint.overrides[1].extends).toEqual([
+        'plugin:@nrwl/nx/javascript',
       ]);
+
+      runCLI(`generate @nrwl/workspace:lib ${mylib} --no-interactive`);
+      // should add new tslint
```

Comment:
nitpick: comment isn't right? was it supposed to be eslint?

Reply:
It's actually not adding anything. I will remove that line